Sources/FingerprintJS/Common/FingerprintStabilityLevel.swift

```diff
@@ -0,0 +1,17 @@
+/// A stability level that indicates which signals are to be included when computing the fingerprint.
```

Comment:
Most of the entities available through the public API are part of the `Library` folder. I'd move this file there for consistency as well.

Reply:
Done.